spec/requests/api/v1/articles_spec.rb

```diff
@@ -0,0 +1,144 @@
+require 'rails_helper'
+
+RSpec.describe 'Articles', type: :request do # rubocop:disable Metrics/BlockLength
+  headers = { 'Authorization': "Token token=#{ENV['HOWITZER_TOKEN']}" }
+
+  describe 'GET #index' do
+    let!(:articles) { create_list(:article, 10) }
+    before { get api_v1_articles_path, headers: headers }
+
+    it 'returns a successful response' do
+      expect(response).to be_successful
+    end
+
+    it 'returns the list of articles' do
+      expect(json.map { |x| x['id'] }).to eq [*1..10]
+    end
+  end
+
+  describe 'GET #show' do
+    let!(:articles) { create_list(:article, 5) }
+    context 'existing article' do
+      before { get '/api/v1/articles/3', headers: headers }
+
+      it 'returns a successful response' do
+        expect(response).to be_successful
+      end
+
+      it 'returns existing article with specified id' do
+        expect(json['id']).to eq 3
+      end
+    end
+
+    context 'not existing article' do
+      it 'raises an error' do
+        assert_raises(ActiveRecord::RecordNotFound) do
+          get '/api/v1/articles/1234', headers: headers
+        end
+      end
+    end
+  end
+
+  describe 'GET #create' do
```

Comment:
Hm. Yeah, bingo! Good example, why GET #create does not make sense from naming perspective.

It is classical REST API with following routes

```
GET /api/v1/articles - returns list of all articles
GET /api/v1/articles/1 - returns article with id 1
POST /api/v1/articles - create a new article
UPDATE /api/v1/articles/1 - updates the article
DELETE /api/v1/articles/1 - removes the article
```

So you not executing something like ArticlesController#get_create. You test it like GET /api/v1/articles  and regarding REST it returns you collection of articles. Please remove extra tests!

Reply:
done